tests/unit/controlgroup/controlgroup_options.js

```diff
@@ -0,0 +1,113 @@
+module( "Controlgroup: options" );
+
+function hasCornerClass( element ) {
+	return !!element.attr( "class" ).match( /ui-corner/g );
+}
+
+test( "disabled", function() {
+	expect( 4 );
+	var element = $( ".controlgroup" ).controlgroup().controlgroup( "option", "disabled", true );
+	strictEqual( element.hasClass( "ui-state-disabled" ), false,
+		"ui-state-disabled is not present on widget after setting disabled true" );
+	strictEqual( element.find( ".ui-state-disabled" ).length, 6,
+		"Child widgets are disabled" );
+
+	element.controlgroup( "option", "disabled", false );
+	strictEqual( element.hasClass( "ui-state-disabled" ), false,
+		"ui-state-disabled is not present on widget after setting disabled false" );
+	strictEqual( element.find( ".ui-state-disabled" ).length, 0,
+		"Child widgets are not disabled" );
+
+});
+test( "items - null", function() {
+	expect( 2 );
+	var element = $( ".controlgroup" ).controlgroup({
+		items: {
+			"button": null,
+			"selectmenu": null,
+			"checkboxradio": null
+		}
+	});
+
+	strictEqual( element.children( ".ui-button" ).length, 0,
```

Comment:
Could also search for `ui-widget` or whatever class is present on all three items properties. Currently you ignore both "selectmenu" and "checkboxradio".